activities.el

```diff
@@ -495,6 +530,59 @@ accordingly."
       (setf activities-mode-timer nil))
     (remove-hook 'kill-emacs-hook #'activities-mode--killing-emacs)))
 
+(require 'easymenu)
```

Comment:
It would be good to add a header comment for this section.

Reply:
Done.